packages/amplify-category-api/src/__tests__/graphql-transformer/api-key-helpers.test.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiKeyConfig } from '@aws-amplify/graphql-transformer-interfaces';
+import { hasApiKey } from '../../graphql-transformer/api-key-helpers';
+
+jest.mock('amplify-cli-core', () => {
+  const original = jest.requireActual('amplify-cli-core');
+  return {
+    ...original,
+    ApiCategoryFacade: {
+      getApiKeyConfig: jest.fn(() => ({
+        apiKeyExpirationDays: 2,
+        apiKeyExpirationDate: new Date('2021-08-20T20:38:07.585Z'),
+        description: '',
+      } as ApiKeyConfig)),
```

Comment:
Is this just temporarily necessary while the CLI hasn't released yet?

Reply:
No, this is just swapping the mock mechanism to mock out the underlying function (getApiKeyConfig) instead of mocking the input data.